### PR TITLE
Fix list of speakers names not displayed

### LINKED ISSUE #3798: Autopilot > List of speakers: Missing names after reload
**Description:** There is a strange error in the autopilot that hides the names of the users. When you reload the autopilot, it can happen that user names are hidden in the list of speakers and in the 'Last speakers' list. If you then hover over the X in the list of recent speakers, for example, the names will reappear. Unfortunately, I cannot say whether there is a system for hiding user names. It seems arbitrary. 

**Example of missing username in last speaker list**
image

**Reproduction:**
1. open the OS4 Demo > Meeting Bundeskonferenz 2024
2. open the Autopilot
3. Reload the page -> Usernames disappear
4. Open 'Last speakers' in Autopilot and hover over the X. Usernames appears again. Hover over the Play-Button -> Names will be visible again. 

Hint: Unfortunately, the problem can only be reproduced cleanly once after calling up the instance. To provoke it again, it sometimes works for me to reload the page with Ctrl+Shift+R. 

**What should happen:**
The list of speakers and the last speakers list should always display the usersnames after reloading the Autopilot.

### PR DESCRIPTION
**Pull request by bastianjoel**
resolves #3798

This PR changes the `_as_observable` suffix to just `$` for view models to comply with angulars [naming conventions for observables](https://v17.angular.io/guide/observables#naming-conventions-for-observables). Besides from that the observable relations do not need to be specified manually anymore. Instead a new type `ViewModelRelations` is used. 

Another change included here is the change of the Shared worker type to `module` to support activating the shared worker while development. This ability was broken with switching to the new angular compiler.

### CODE REVIEW

**Review by MSoeb — CHANGES_REQUESTED** [review]
Good: The names always loads correctly after several reloads.

Bad: The performance issue still occurs.

**Comment by bastianjoel** [comment]
The performance issues are not cause by the client

File: client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-entry/list-of-speakers-entry.component.html [hunk 1]
```diff
@@ -234,7 +235,7 @@
                 </div>
             </div>
             <ng-container *osPerms="permission.listOfSpeakersCanManage">
-                @if (speaker.speech_state === SpeechState.INTERPOSED_QUESTION && !speaker.meeting_user_id) {
```
Comment (luisa-beerboom) [inline client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-entry/list-of-speakers-entry.component.html:237]: Bad. Revert this line.

**Review by luisa-beerboom — APPROVED** [review]
Nice

**Review by MSoeb — APPROVED** [review]
looks fine

**Pull request merged**